app/utils/snyk_helper.py

```diff
@@ -183,6 +183,12 @@ def import_manifests(org_id, repo_full_name, integration_id, files=[]) -> Import
     path = f"org/{org.id}/integrations/{integration_id}/import"
 
     if len(files) > 0:
+        # verify against set limit per repo
+        if len(files) > common.MAX_IMPORT_MANIFEST_PROJECTS:
+            raise snyk.errors.SnykError(
```

Comment:
instead of raising an error here and exiting the application, what if we attempt to import the up to the `common.MAX_IMPORT_MANIFEST_PROJECTS` 

e.g. `files[:common.MAX_IMPORT_MANIFEST_PROJECTS]`

otherwise, nothing will get imported.

Reply:
let's list the manifests that will not get imported (beyond the MAX number) into a log file, similar to how the other ones are setup.  We can tell the user on the screen that not all manifests will be imported because its more than max, and to check the file for details about which ones will not be imported.

Reply:
skipped manifests above the limit are now logged to a CSV file.